backend/molgenis-emx2-io/build.gradle

```diff
@@ -5,4 +5,9 @@ dependencies {
     implementation project(':backend:molgenis-emx2-tasks')
     testImplementation project(':backend:molgenis-emx2-sql')
     testImplementation project(':backend:molgenis-emx2-datamodels')
+    //lombok
```

Comment:
Why would we need this ? Java has records now , lombok introduces an additional code generation step, that may not play nice with other java features, do we really really need this ? , the google autoclasses in emx1 causes al kinds of issues

Reply:
My experiences with records are that they also cause issues. Moreover, I am fan of builder pattern and lombok provides that out of box. Advantage is that it is systematic (as compared to manually building). 

I also felt hesitant to add a generator but lombok really works suprisingly well (and I found it easier then AutoValue). So we don't *need* it but I found myself generating code in Intellij and then changing it which is also a maintenance time sink.

Reply:
P.S. I think Java shouldn't have created 'record' but instead have integrated lombok ;-)

Reply:
This argument puts writing code ahead of reading and refactoring code, i see this as a mistake. This puts the burden on the maintainers instead of the creator. Once you add a lib like lombok its going to be very hard to go back and you are going to writing a dialect of java ( same holds for framework like spring) instead of java. i would urge to reconsider for such minor gains.

Reply:
"I got serialization issues in JSON" what type of issue ?

Reply:
> "I got serialization issues in JSON" what type of issue ?

If I remember correctly jackson was not able to instantiate records, but I have to check again. FYI I started with a record before moving to lombok. I am totally happy to simply generate the bean as we have done before :-)